x-pack/auditbeat/processors/sessionmd/procfs/procfs.go

```diff
@@ -186,6 +187,11 @@ func (r ProcfsReader) GetProcess(pid uint32) (ProcessInfo, error) {
 	return r.getProcessInfo(proc)
 }
 
+func (r ProcfsReader) ProcessExists(pid uint32) bool {
```

Comment:
The receiver isn't used (kind of surprising) so we can drop the `r`. I figured ProcfsReader might hold an instance of https://pkg.go.dev/github.com/prometheus/procfs#FS to handle cases where it was looking at an alternative procfs mount point.

```suggestion
func (ProcfsReader) ProcessExists(pid uint32) bool {
```